python/insights_ansible_playbook_verifier/app.py

```diff
@@ -0,0 +1,113 @@
+import argparse
+import logging
+import pathlib
+import pkgutil
+import sys
+
+import insights_ansible_playbook_lib as lib
+
+logger = logging.getLogger(__name__)
+
+
+def read_playbook_from_stdin() -> str:
+    """Read playbook with to-be-verified plays from standard input."""
+    try:
+        result: str = "".join(line for line in sys.stdin)
+    except KeyboardInterrupt:
+        logger.error("Received keyboard interrupt.")
+        raise RuntimeError("Playbook loading was aborted.") from None
+    return result
+
+
+def read_revocation_playbook_from_package() -> str:
+    """Read revocation playbook content saved in the package."""
+    data: str = pkgutil.get_data(
+        "insights_ansible_playbook_verifier",
+        "data/revoked_playbooks.yml",
+    ).decode("utf-8")  # type: ignore
+    return data
+
+
+def get_gpg_key_content() -> bytes:
```

Comment:
i'd add a `_from_package` suffix to the name of this function, for consistency with `read_revocation_playbook_from_package()`; this way, IMHO it is clearer from where the function reads